docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+# This docker compose definition will create application containers for all of
+# the components of the task-tracker application
+#
+# Usage: docker-compose up --build
+
+version: '3'
+services:
+  backend:
+    build: .
+    # TODO also defined in the dockerfile - where should they live?
+    entrypoint: poetry run flask run
+    environment:
+      # TODO also defined in the dockerfile - where should they live?
```

Comment:
I personally like to make the dockerfile as unopinionated as possible, and put the opinion into the compose file.